### DISCUSSION TITLE
240316

### DISCUSSION THREAD

**Discussion by jisunipark** [codeit-part4-8team-project/main#52] [팀회의]
## 멘토님 질문 사항

**Comment by jisunipark** [comment]
## 논의사항
- sideBar 디렉토리명 변경

## 어제 한 일
- sideBar 디렉토리명 오류 해결
- 코드 스타일에 맞추어 수정

## 오늘 할 일
- 리액트 훅 폼 공부

## 오늘의 일정 (코어 타임)

## 멘토님 질문 사항

**Comment by sobokki** [comment]
## 논의사항

## 어제 한 일
코드 리팩토링( 캘린더 코드에서 따로 뺄 수 있는 부분 분리),(for문 -> Array.from배열로 바꾸기)
## 오늘 할 일
코드 ,useCallback으로 바꿔서 =>최적화 시키기,
## 오늘의 일정 (코어 타임)
## 멘토님 질문 사항

**Comment by MoonPillGyeom** [comment]
## 논의사항
xx
## 어제 한 일 
그룹편집창 만들기
## 오늘 할 일 
제어컴포넌트만들기
## 오늘의 일정 (코어타임)
x
## 멘토님 질문 사항
x
